cg/constants/lims.py

```diff
@@ -12,6 +12,7 @@
     "comment": "Comment",
     "control": "Control",
     "concentration": "Concentration (nM)",
+    "concentration_ng_ul": "Concentration (ng/ul)",
```

Comment:
We should refactor this to a pydantic model in another PR

Reply:
I would like to refactor quite a bit in the LIMS module...